CONTRIBUTING.md

```diff
@@ -139,7 +142,14 @@ To send us a pull request, please:
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
-### Importing icons from other open source repos
+## CI artifact
+
+Each commit and pull request is processed by an automated system which runs
+all tests and provides the build result via the _Details_ link as shown below.
+
+<img src="./docs/images/ci-artifact.png" alt="CI artifact" width="512"/>
```

Comment:
Why is this not using the markdown convention? (examples in extension-readme.md)

Reply:
can't set width that way (except in some flavors of markdown, but not github markdown)

Reply:
Is the width that important? Would it make more sense to just resize on your local machine?

Reply:
Width is important enough to use the `<img>` tag, which is a vanishingly small cost.

I did resize the image itself, but the image editor disagrees with the web browser on what a width of 512 pixels looks like. In general also, having to resize the image physically is pretty awkward way to adjust layout in a readme file. Using `<img ... width=...>` is a reasonable practice.